meson.build

```diff
@@ -0,0 +1,1629 @@
+    dependencies += jack
+    sources += 'audio/out/ao_jack.c'
+endif
+
+openal = dependency('openal', version: '>= 1.13', required: get_option('openal'))
+if openal.found()
+    dependencies += openal
+    sources += 'audio/out/ao_openal.c'
+endif
+
+opensles = {
+    'name': 'opensles',
+    'deps': cc.find_library('OpenSLES', required: get_option('opensles')),
+    'statement': run_command('cat', 'waftools/fragments/opensles.c').stdout(),
+    'use': false,
+}
+if opensles['deps'].found()
+    opensles += {'use': cc.compiles(opensles['statement'], dependencies: opensles, name: 'opensles check')}
+endif
+if opensles['use']
+    dependencies += opensles['deps']
+    sources += 'audio/out/ao_opensles.c'
+endif
+
+oss = {
+    'name': 'oss',
+    'statement': run_command('cat', 'waftools/fragments/oss.c').stdout(),
+    'use': false,
+}
+if not get_option('oss-audio').disabled()
+    oss += {'use': cc.compiles(oss['statement'], name: 'oss check')}
+endif
+if oss['use']
+    sources += 'audio/out/ao_oss.c'
+endif
+
+pulse = dependency('libpulse', version: '>= 1.0', required: get_option('pulse'))
+if pulse.found()
+    dependencies += pulse
+    sources += 'audio/out/ao_pulse.c'
+endif
+
+wasapi = {
+    'name': 'wasapi',
+    'statement': run_command('cat', 'waftools/fragments/wasapi.c').stdout(),
+    'use': false,
+}
+if not get_option('wasapi').disabled() and win32
+    wasapi += {'use': cc.compiles(wasapi['statement'], name: 'wasapi check')}
+endif
+if wasapi['use']
+    sources += ['audio/out/ao_wasapi.c',
+                'audio/out/ao_wasapi_changenotify.c',
+                'audio/out/ao_wasapi_utils.c']
+endif
+
+
+# video output dependencies
+caca = dependency('caca', version: '>= 0.99.beta18', required: get_option('caca'))
+if caca.found()
+    dependencies += caca
+    sources += 'video/out/vo_caca.c'
+endif
+
+drm = {
+    'name': 'drm',
+    'deps': dependency('libdrm', version: '>= 2.4.74', required: get_option('drm')),
+    'header': vt_h['use'] or consio_h['use'],
+}
+drm += {'use': drm['deps'].found() and drm['header']}
+if drm['use']
+    dependencies += drm['deps']
+    sources += ['video/out/drm_atomic.c',
+                'video/out/drm_common.c',
+                'video/out/drm_prime.c',
+                'video/out/opengl/hwdec_drmprime_drm.c',
+                'video/out/vo_drm.c']
+endif
+
+gbm = {
+    'name': 'gbm',
+    'deps': dependency('gbm', required: get_option('gbm')),
+    'header': cc.check_header('stdio.h') and cc.check_header('gbm.h'),
+}
+gbm += {'use': gbm['deps'].found() and gbm['header']}
+if gbm['use']
+    dependencies += gbm['deps']
+endif
+
+jpeg = cc.find_library('jpeg', required: get_option('jpeg'))
+if jpeg.found()
+    dependencies += jpeg
+endif
+
+libplacebo_project = subproject(
+    'libplacebo',
+    required: false,
+    version: '>3.120.3'
+)
+libplacebo = dependency('libplacebo', fallback: ['libplacebo', 'libplacebo'], version: '>=2.72.0', required: get_option('libplacebo'))
```

Comment:
Adding a subproject() and then doing dependency with fallback, isn't the correct way to handle this. :)

You don't even *have* a subproject available to try configuring... or perhaps you do locally, but never added it to the PR.

But the neat thing about subprojects is you don't need to manually `subproject()` them if you just want dependency fallbacks. See https://mesonbuild.com/Wrap-dependency-system-manual.html and add one single ini file `libplacebo.wrap` which contains a `[provide]` section. If meson cannot find a copy of libplacebo on the system, it will download the libplacebo tarball, checksum it, extract it, and use it.

If you pass `meson setup builddir/ --force-fallback-for=libplacebo`, it will skip system dependency resolution and use the wrap. Same goes for `meson setup builddir --wrap-mode=forcefallback` which forces *all* wraps to be used as the preferred way to resolve a dependency.

tl;dr Wearing my official "meson core maintainer" hat, @h0tc0d3 is correct and your libplacebo handling is buggy.

This is independent of whether or not --auto-features=enabled is a good idea (and for the record I am not personally fond of that option *and* I believe the arch-meson script should not be used even for distro packaging, for this and a couple of other reasons).